cmd/fablab/subcmd/list.go

```diff
@@ -86,3 +88,52 @@ func listModels(_ *cobra.Command, _ []string) {
 	}
 	fmt.Println()
 }
+
+var listHostsCmd = &cobra.Command{
```

Comment:
The `fablab list` commands are all related to user-environment details (instances, models), not model details (hosts).

Reply:
I could add a `model` sub-command, so it would be `fablab model list hosts`. Let me know if that sounds good, or if you'd prefer another path.

Reply:
What if we made it a `dump` subcommand, and only JSON dumped the parts of the model that the subcommand shows? `fablab dump hosts` for example? I kind of like the idea of making the output machine parseable, but also human parseable.

If we do end up with a duplication of the `dump` subcommand in a format that's explicitly human parseable, it'd be nice to incorporate some sort of tabular formatting library or some such, so that we don't end up with one-off output looks all over the place.

Reply:
I think having both is probably useful. I suspect I'll find it less useful as I use fablab more, but for now being able to see what hosts I have is very helpful.

I'll take a look at adding it to dump and look into formatting libs.

Reply:
👍